source/views/student-orgs/list.js

```diff
@@ -173,6 +184,43 @@ export class StudentOrgsView extends React.Component {
     })
   }
 
+  splitToArray = (str: string) => {
```

Comment:
At some point (not this PR!) we should really pull this out into its own function.

And, like, make some documentation of those functions.